plugins/woocommerce-blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/block-settings/index.tsx

```diff
@@ -53,7 +49,6 @@ export const ProductGalleryThumbnailsBlockSettings = ( {
 	const maxNumberOfThumbnails = 8;
 	const minNumberOfThumbnails = 3;
 	const { productGalleryClientId } = context;
-	// @ts-expect-error @wordpress/block-editor/store types not provided
 	const { updateBlockAttributes } = useDispatch( blockEditorStore );
 
 	return (
```

Comment:
The `<Icon>` components are being flagged by the linter for missing attributes. We could fix this I think by just adding them with undefined values like so:

```tsx
<Icon
	size={ 32 }
	icon={ thumbnailsPositionLeft }
	onPointerEnterCapture={ undefined }
	onPointerLeaveCapture={ undefined }
/>
```

Reply:
I don't think that's the right approach. I'm against adding unnecessary properties to satisfy linter/TS. I realised in most places we use Icon component from `@wordpress/components` and not `@wordpress/icons` like in here. So I replaced them and the linter issues are gone 👍